matsuoharuki/pbl2_group2
# Show spots in user page

### Updated
- When you register a spot, DB also keeps your user ID.
- You can see spots which you registered in your user page.

### How to test
1. Log in the App.
1. Post some spots.
1. Jump to your user page, push hamburger button, and push "作成スポット". You can see the spots you registered.
### Related issue
#97
